### ISSUE TITLE
Lingua suddenly not working

### ISSUE THREAD

**Issue by bennettpeter** [chr33z/vscode-lingua#81]
I upgraded VS Code (automatic upgrade in xubuntu)
Version: 1.77.3
Commit: 704ed70d4fd1c6bd6342c436f1ede30d1cff4710
Date: 2023-04-12T09:16:52.732Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Linux x64 5.15.0-69-generic
Sandboxed: Yes

Lingua options no longer show in right click menu, translations do not show in the editor. I reinstalled Lingua, that did not help.
Lingua: Select default language gives error command:lingua.selectDefaultLanguage not found.

**Comment by bennettpeter** [comment]
I tried installing prior versions of VS Code and lingua still does not work, so it is not the version of VS Code that has caused the problem. Something is causing lingua to stop working, and I cannot figure out what it is.

**Comment by chr33z** [comment]
Hey Peter,

thanks for your feedback! On my test setup, lingua is working as intended on the newest vscode release 1.77.3.

In order to analyse the error, it would be helpful if you could provide a sample project, that does not work for your setup. Are there any messages in "Debug Console" that would indicate an error?

Lingua does not depend on any special libraries, but maybe the project setup is as such that the extension can not recognize the project and therefore not load the lingua features.

Best,
Christopher

**Comment by bennettpeter** [comment]
This morning I woke my laptop and opened vs code, opened the project and it is working.

The whole day Saturday and Sunday it would not work, even with all the things I tried, reinstalling, etc. Now this morning I did not even reboot and it is all working.

The debug console is empty. If it happens again I will make sure to look there.

Sorry for the trouble, and thanks for the quick response.

Thank you for a great product.

Regards,
Peter
